src/App.vue

```diff
@@ -458,76 +386,98 @@
     </div>
     <!-- Main Content End -->
 
-    <!-- Footer Section Start -->
-    <footer>
-      <div class="container">
-        <div class="row">
-          <div class="col-md-12">
-            <div class="social-icons">
-              <ul>
-                <li class="facebook">
-                  <a href="#">
-                    <i class="fa fa-facebook" />
-                  </a>
-                </li>
-                <li class="twitter">
-                  <a href="#">
-                    <i class="fa fa-twitter" />
-                  </a>
-                </li>
-                <li class="google-plus">
-                  <a href="#">
-                    <i class="fa fa-google-plus" />
-                  </a>
-                </li>
-                <li class="linkedin">
-                  <a href="#">
-                    <i class="fa fa-linkedin" />
-                  </a>
-                </li>
-                <li class="pinterest">
-                  <a href="#">
-                    <i class="fa fa-pinterest" />
-                  </a>
-                </li>
-                <li class="dribbble">
-                  <a href="#">
-                    <i class="fa fa-dribbble" />
-                  </a>
-                </li>
-              </ul>
+    <!-- cart -->
+    <div id="cart" class="col-sm-3 bg-gray-light d-none d-md-block">
+      <div class="m-4 text-center">
+        <h2 class="head-title d-inline-block font-weight-bold text-orange-bright">
+          CART
+        </h2>
+      </div>
+      <ul class="list-group list-group-flush">
+        <li
+                v-show="selectedActivity.id"
+                class="list-group-item"
+        >
+          <div class="row d-flex justify-content-between lead">
+            <div class="col-sm-6">
+              {{ selectedActivity.name }}
+            </div>
+            <!--<div class="col-sm-6 pull-right">-->
+            <!--¥ 0.00-->
+            <!--</div>-->
+          </div>
+        </li>
+        <li
+                v-if="selectedPeople.adults"
+                class="list-group-item"
+        >
+          <div class="row d-flex justify-content-between lead">
+            <div class="col-sm-6">
+              Adults
             </div>
-            <div class="site-info">
-              <p>
-                All copyrights reserved &copy; 2017 - Designed & Developed by <a
-                  rel="nofollow"
-                  href="https://uideck.com"
-                >
-                  UIdeck
-                </a>
-              </p>
+            <div class="col-sm-6 pull-right">
+              ¥ {{ activityCosts.adults }}
```

Comment:
This should be made dynamic as well depending on `selectedPeople`. Leave it as is for now and create a ticket for in Trello